templates/100.html

```diff
@@ -0,0 +1,24 @@
+{% extends '_base.html' %}
+{% block content %}
+
+  <div class="row mt-3">
+    <div class="col">
+        <h1>100 theorems</h1>
+        
+        <p><a href="https://www.cs.ru.nl/~freek/">Freek Wiedijk</a> has created <a href="https://www.cs.ru.nl/~freek/100/">a list</a> of 100 theorems as targets for formalization as a way of tracking and comparing prominent theorem provers.
+        The progress on problems on this list in Lean is as follows:</p>
+
+  <div class="list-group">
+    {% for theorem in hundred_theorems %}
+    {% if theorem.link or theorem.decl or theorem.author or theorem.note %}
+    <div class="list-group-item">
+        <h5 class="card-title">{{ theorem.number }}: {{ theorem.title }}</h5><h6 class="card-title">Author: {{ theorem.author }}</h5>
```

Comment:
```suggestion
        <h5 class="card-title markdown-heading" id="{{ theorem.number }}">{{ theorem.number }}: {{ theorem.title }} <a class="hover-link" href="#{ theorem.number }">#</a></h5>
        <h6 class="card-title">Author: {{ theorem.author }}</h6>
```
This should make the theorems linkable (following #11). I also fixed what looks to be a typo: `</h5>` -> `</h6>`